modules/nodes-cluster-enabling-features-console.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-cluster-enabling-features.adoc
+
+[id="nodes-cluster-enabling-features-console_{context}"]
+= Enabling feature sets using the web console
+
+You can use the {product-title} web console to enable feature sets for all the nodes in a cluster by editing the `FeatureGate` custom resource (CR).
+
+.Procedure
+
+To enable feature sets:
+
+. In the {product-title} web console, switch to the *Administration* -> *Custom Resource Definitions* page.
+
+. On the *Custom Resource Definitions* page, click *FeatureGate*.
+
+. On the *Custom Resource Definition Details* page, click the *Instances* tab.
+
+. Click the *cluster* feature gate, then click the *YAML* tab.
+
+. Edit the *cluster* instance to add specific feature sets:
++
+.Sample Feature Gate custom resource
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: FeatureGate
+metadata:
+  name: cluster <1>
+....
+
+spec:
+  featureSet: TechPreviewNoUpgrade <2>
+----
++
+--
+<1> The name of the `FeatureGate` CR must be `cluster`.
+<2> Add the feature sets that you want to enable in a comma-separated list:
+* `IPv6DualStackNoUpgrade` to enable the dual-stack networking mode.
+* `TechPreviewNoUpgrade` to enable specific Technology Preview features.
```

Comment:
Small nit for these two bullets:
* `IPv6DualStackNoUpgrade` enables the dual-stack networking mode.
* `TechPreviewNoUpgrade` enables specific Technology Preview features.